### PR TITLE
[legolandholidays.co.uk] New ruleset

### PR DESCRIPTION
**Pull request by youdly**
Mainly for legolandholidays.co.uk, other 2 rulesets are to secure mixed content of that website.

### CODE REVIEW

File: src/chrome/content/rules/Merlinbreaks.co.uk.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	<target host="*.merlinbreaks.co.uk" />
+
+		<test url="http://chessingtonholidays.merlinbreaks.co.uk/" />
+		<test url="http://legoland.merlinbreaks.co.uk/" />
+
+        <!-- cert mismatch -->
+	<exclusion pattern="^http://(www\.)?merlinbreaks\.co\.uk/" />
+
+		<test url="http://www.merlinbreaks.co.uk/" />
+		<test url="http://merlinbreaks.co.uk/" />
```
Comment (J0WI) [inline src/chrome/content/rules/Merlinbreaks.co.uk.xml]: refused host

File: src/chrome/content/rules/HolidayExtras.com.xml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<!--
+	Problematic subdomain:
+		- holidayextras.com				(hostname mismatch, CN: www.hxtrack.com)
+		- press-office.holidayextras.co.uk		(hostname mismatch, CN: *.pagely.com, pagely.com)
+-->
+<ruleset name="HolidayExtras.com">
```
Comment (J0WI) [inline src/chrome/content/rules/HolidayExtras.com.xml]: WFM:
hungrygeek.holidayextras.co.uk
hxtrack.holidayextras.co.uk
app.holidayextras.co.uk

CloudFlare error on `brand.holidayextras.com`

Mismatch:
mobile.holidayextras.co.uk
join.holidayextras.co.uk
docs.holidayextras.co.uk

**Comment by J0WI** [comment]
Posted a few more subdomains.
Lgtm then.

**Comment by youdly** [comment]
Added subdomains.

**Pull request merged**